features/dishes_sorted_in_categories.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Visitor can see dishes sorted in categories
+    As a visitor,
+    In order to navigate the products
+    I would like to see the products sorted in categories
+
+    Scenario: View list of dishes sorted in categories
+        Given the following dishes exist
+        | name     | description          | price | category |   
+        | olives   | green little ones    | 20    | starter  |
+        | pizza    | italian favorite     | 100   | main     |
+        | cake     | homemade             | 30    | dessert  |
+
```

Comment:
Im a bit skeptical of this feature test, I would like you to post a screenshot of how this page looks with all of the products listed in categories

Reply:
Please find it below. We are displaying the category names right now for testing purposes. But once we start working on the final view implementation of the index page, the intention is to hide the category names and instead section off the page according to the category attributes of each dish.

![image](https://user-images.githubusercontent.com/43792384/49173636-8f29b100-f344-11e8-8c72-d1bba92d7df6.png)

Reply:
One of the tasks on this feature is `List all products within their category on the product index page`.
The current implementation of this feature does not do that.
In your view you are not listing the products within their categories, you are showing the category for each product. 

What I would like to see is you to change the feature test steps to say:
`Then I should see "olives" in the "starter" category`

This would also mean that you would have to change the way you present the categories and dishes on the index page.

Reply:
Absolutely, point taken. We've now corrected that. Thanks.